src/python/pants/backend/javascript/subsystems/nodejs_test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+import logging
+
+import pytest
+
+from pants.backend.javascript.subsystems import nodejs
+from pants.backend.javascript.target_types import JSSourcesGeneratorTarget
+from pants.backend.python import target_types_rules
+from pants.core.util_rules import config_files, source_files
+from pants.engine.process import ProcessResult
+from pants.testutil.rule_runner import QueryRule, RuleRunner
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def rule_runner() -> RuleRunner:
+    return RuleRunner(
+        rules=[
+            *nodejs.rules(),
+            *source_files.rules(),
+            *config_files.rules(),
+            *target_types_rules.rules(),
+            QueryRule(ProcessResult, [nodejs.NpxProcess]),
+        ],
+        target_types=[JSSourcesGeneratorTarget],
+    )
+
+
+def test_npx_process(rule_runner: RuleRunner):
+    rule_runner.set_options(
+        [
+            "--backend-packages=['pants.backend.javascript']",
+        ],
+        env_inherit={"PATH", "PYENV_ROOT", "HOME"},
```

Comment:
I don't think these should be necessary. Definitely not the backend package, and I doubt PYENV_ROOT

Reply:
Ah, probably copy/paste from another test.